src/olympia/files/views.py

```diff
@@ -98,8 +97,7 @@ def browse(request, viewer, key=None, type='file'):
     data['poll_url'] = reverse('files.poll', args=[viewer.file.id])
     data['form'] = form
 
-    if (not waffle.switch_is_active('delay-file-viewer') and
-            not viewer.is_extracted()):
+    if not viewer.is_extracted():
         extract_file(viewer)
```

Comment:
The thing is, the 'delay-file-viewer' flag isn't set on prod or anywhere and possibly never was. So we're not using async extraction at the moment (unless I'm missing something) and I found, if it's not used, let's remove it until we might rethink the way we do it and then do it right